services/hacker.service.js

```diff
@@ -64,9 +64,25 @@ async function findIds(queries) {
     return ids;
 }
 
+/**
+ * @function findByAccountId
+ * @param {ObjectId} accountId
+ * @return {DocumentQuery[]} A list of hacker document queries found with the accountId
+ */
+function findByAccountId(accountId) {
+    const TAG = `[ Hacker Service # findByAccountId ]:`;
+
+    const query = {
+        accountId: accountId
+    };
+
+    return Hacker.find(query, logger.updateCallbackFactory(TAG, "hacker"));
```

Comment:
We could do `findOne` here, since we know that `accountId` is unique. Therefore, we don't need to return a list of  `DocumentQuery`s

Reply:
Updated